orchestrator-bundle/build.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+function build() {
+    charm=$1
+    cd $charm/
+    charmcraft pack
+    mv ${charm}_ubuntu-20.04-amd64.charm $charm.charm
```

Comment:
```suggestion
    charm="$1"
    pushd "${charm}/"
    charmcraft pack
    mv "${charm}_ubuntu-20.04-amd64.charm" "${charm}.charm"
    popd
```

Using `cd` in a script like this can be a little tough to keep track of -- using pushd/popd within a function means you'll always finish up where you left off :)

Reply:
Agreed, fixed